integration-boilerplate/src/backend/backend.ts

```diff
@@ -0,0 +1,128 @@
+import { Router, Request, Response } from 'express';
+import { jweInit, jwsInit } from '../answers-integration-utils';
+import { getInjectedIntegrationScript } from '../script';
+import * as bodyParser from 'body-parser';
+import { MongoWrapper } from './database';
+
+export type AnswersApiConfig = {
+	answersApiSecret: string
+	integrationId: string,
+	baseUrl: string,
+	ecryptKey: string,
+	apiPort: number,
+	routes: {
+		path: string,
+		register: string,
+		unregister: string,
+		settings: string,
+		script: string,
+		view: string,
+	},
+	mongo: {
+		mongoUrl: string,
+		initDataDB: string,
+		settingsDB: string,
+	}
+};
+
+export const initAnswersApi = async (app: Router, dbWrapper: MongoWrapper, config: AnswersApiConfig) => {
+	const ANSWERS_PATH = config.routes.path;
+
+	const jweInstance = await jweInit(config.answersApiSecret);
+	const jwsInstance = await jwsInit(config.answersApiSecret);
+
+	app.use(bodyParser.urlencoded({ extended: true }));
+	app.use(bodyParser.json());
+
+	app.post(`/${ANSWERS_PATH}/${config.routes.register}`, async (req: Request, res: Response) => {
```

Comment:
I think that for simplicity the paths should be inlined:
`/integration/register`
`/integration/unregister`
etc

I think it'll make the config simpler and make it more readable